code/datums/mode/abductors.dm

```diff
@@ -0,0 +1,76 @@
+/datum/role/abductor //Generic, not for direct use
+	name = "abductor"
+	antag_flag = ROLE_ABDUCTOR
+	threat = 2 //wip
+
+/datum/role/abductor/equip()
+	if(!ishuman(owner.current))
+		return
+	var/mob/living/carbon/human/H = owner.current
+	var/radio_freq = SYND_FREQ
```

Comment:
yep. Whoever designed abductors gave them syndicate headsets to let to hear everything.

Keep in mind that abductors can't talk, so they can't shittalk nuke ops.